network/host.go

```diff
@@ -187,9 +187,7 @@ func (h *host) send(p peer.ID, msg Message) (err error) {
 		return err
 	}
 
-	// append leb128 variable-length encoding
-	encMsg = encodeMessageLEB128(encMsg)
-
+	// TODO: append leb128 variable-length encoding
```

Comment:
reopened #484

Reply:
pls add the issue number to the comment so it can be tracked back.